src/components/Navigation.tsx

```diff
@@ -26,17 +26,35 @@ const Navigation: React.FC<Props> = ({
   const [scrollTop, setScrollTop] = useState(0);
   const scrollThreshold = 10;
 
+  const [navBarBgColor, setNavBarBgColor] = useState(false);
```

Comment:
This will only change between white and grey, so can we name it more precise. Something like `hasWhiteBackground`?